src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Figure: ")
```

Comment:
The message in the `draw` method should be in English and the text 'Figure: ' should be followed by the actual figure type, e.g., 'Square'.